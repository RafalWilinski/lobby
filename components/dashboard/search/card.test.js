import React from "react";
import renderer from "react-test-renderer";
import Card from "./Card";

describe("Card", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Card title="title" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
