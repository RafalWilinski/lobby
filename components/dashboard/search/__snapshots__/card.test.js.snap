// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="ant-card ant-card-bordered"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="ant-card-head"
  >
    <div
      className="ant-card-head-title"
    >
      title
    </div>
  </div>
  <div
    className="ant-card-body"
    style={undefined}
  >
    <div
      style={
        Object {
          "display": "flex",
        }
      }
    >
      <img
        src="http://stamfordresearch.com/wp-content/uploads/2016/06/opencv-950x591.jpg"
        style={
          Object {
            "height": "100px",
          }
        }
      />
      <div
        style={
          Object {
            "marginLeft": "20px",
          }
        }
      >
        <p>
          Lorem Ipsum jest tekstem stosowanym jako przykładowy wypełniacz w przemyśle poligraficznym. Został po raz pierwszy użyty w XV w. przez nieznanego drukarza do wypełnienia tekstem próbnej książki. Pięć wieków później zaczął być używany przemyśle elektronicznym, pozostając praktycznie niezmienionym.
        </p>
        <h3
          style={
            Object {
              "marginTop": "10px",
            }
          }
        >
          Zespół
        </h3>
        <div>
          <img
            onBlur={undefined}
            onClick={undefined}
            onFocus={undefined}
            onMouseDown={undefined}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={undefined}
            src="http://rwilinski.me/static/profile.67d1d64d.png"
            style={
              Object {
                "borderRadius": "50%",
                "boxShadow": "1px 2px 2px 1px #aaaaaa",
                "height": "40px",
                "marginRight": "-8px",
              }
            }
          />
          <img
            onBlur={undefined}
            onClick={undefined}
            onFocus={undefined}
            onMouseDown={undefined}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={undefined}
            src="https://media-exp2.licdn.com/media/AAEAAQAAAAAAAApoAAAAJDQ1NmYxMjljLWQxNGQtNDI5ZS05NjRlLThjMDg4MDU4YjgzYQ.jpg"
            style={
              Object {
                "borderRadius": "50%",
                "boxShadow": "1px 2px 2px 1px #aaaaaa",
                "height": "40px",
                "marginRight": "-8px",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
